src/gui/tray/FileActivityDialog.qml

```diff
@@ -0,0 +1,17 @@
+import QtQuick.Window 2.15
+
+import com.nextcloud.desktopclient 1.0 as NC
+
+Window {
+    id: dialog
+
+    property NC.FileActivityListModel model: NC.FileActivityListModel { }
```

Comment:
better do that
```
property alias model: activityModel

NC.FileActivityListModel {
    id: activityModel
}
```
I know this is more verbose but also much more usual way to declare an item